### PR TITLE
Update KMS provider page

### PR DESCRIPTION
**Pull request by tengqm**
This PR updates the KMS provider page by:

- applying format tweaking for long lines, missing empty lines etc.
- adding a link to the config API where users may find more details.

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/kms-provider.md [hunk 1]
```diff
@@ -94,32 +112,43 @@ To encrypt the data:
          - identity: {}
    ```
 
-1. Set the `--encryption-provider-config` flag on the kube-apiserver to point to the location of the configuration file.
+1. Set the `--encryption-provider-config` flag on the kube-apiserver to point to
+   the location of the configuration file.
 1. Restart your API server.
 
+For details about the `EncryptionConfiguration` format, please check
+[API server encryption API reference](/docs/reference/config-api/apiserver-encryption.v1/).
```
Comment (tengqm) [inline content/en/docs/tasks/administer-cluster/kms-provider.md:120]: This is the sentence added. Other changes in this PR are all about markdown format.

File: content/en/docs/tasks/administer-cluster/kms-provider.md [hunk 2]
```diff
@@ -19,43 +19,58 @@ This page shows how to configure a Key Management Service (KMS) provider and plu
...
 * `name`: Display name of the KMS plugin.
 * `endpoint`: Listen address of the gRPC server (KMS plugin). The endpoint is a UNIX domain socket.
 * `cachesize`: Number of data encryption keys (DEKs) to be cached in the clear.
   When cached, DEKs can be used without another call to the KMS;
   whereas DEKs that are not cached require a call to the KMS to unwrap.
-* `timeout`: How long should kube-apiserver wait for kms-plugin to respond before returning an error (default is 3 seconds).
+* `timeout`: How long should `kube-apiserver` wait for kms-plugin to respond before
+  returning an error (default is 3 seconds).
 
 See [Understanding the encryption at rest configuration.](/docs/tasks/administer-cluster/encrypt-data)
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/kms-provider.md:42]: Extra tidying opportunity:
```suggestion
See [Understanding the encryption at rest configuration](/docs/tasks/administer-cluster/encrypt-data).
```

File: content/en/docs/tasks/administer-cluster/kms-provider.md [hunk 3]
```diff
@@ -94,32 +112,43 @@ To encrypt the data:
          - identity: {}
    ```
 
-1. Set the `--encryption-provider-config` flag on the kube-apiserver to point to the location of the configuration file.
+1. Set the `--encryption-provider-config` flag on the kube-apiserver to point to
+   the location of the configuration file.
 1. Restart your API server.
 
+For details about the `EncryptionConfiguration` format, please check
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/kms-provider.md:119]: (nit)
```suggestion
For details about the `EncryptionConfiguration` format, please check the
```

**Review by sftim — COMMENTED** [review]
/lgtm
/approve

/hold
for tweaks - if you wish to @tengqm

**Comment by tengqm** [comment]
/hold cancel
Tweaks applied.

**Pull request merged**